proxy/hdrs/HdrTest.cc

```diff
@@ -496,7 +496,7 @@ HdrTest::test_mime()
     "continuation: part1\r\n"
     " part2\r\n"
     "scooby: doo\r\n"
-    "scooby : doo\r\n"
```

Comment:
I think there's a way to have headers that should fail the parsing too, right? Maybe move the failure one there?